alacritty_terminal/src/config/window.rs

```diff
@@ -1,5 +1,7 @@
 use serde::Deserialize;
 
+use std::os::raw::c_ulong;
```

Comment:
Std imports should be at the top of all imports.

Reply:
Good call, fixing it